src/libstore-c/nix_api_store.h

```diff
@@ -245,6 +245,30 @@ void nix_derivation_free(nix_derivation * drv);
  */
 nix_err nix_store_copy_closure(nix_c_context * context, Store * srcStore, Store * dstStore, StorePath * path);
 
+/**
+ * @brief Gets the closure of a specific store path
+ *
+ * @note The callback borrows each StorePath only for the duration of the call.
+ *
+ * @param[out] context Optional, stores error information
+ * @param[in] store nix store reference
+ * @param[in] store_path The path to compute from
+ * @param[in] flip_direction
```

Comment:
@roberth is the right thing to just copy the doc from the C++ function?

Reply:
If the docs are good, then that's fine